src/ray/core_worker/transport/direct_actor_transport.cc

```diff
@@ -219,6 +221,25 @@ void CoreWorkerDirectTaskReceiver::SetMaxActorConcurrency(int max_concurrency) {
   }
 }
 
+void CoreWorkerDirectTaskReceiver::SetActorAsAsync() {
+  if (!is_async_) {
+    RAY_LOG(DEBUG) << "Setting direct actor as async, creating new fiber thread";
+
+    // Enable current thread to share work
```

Comment:
Prefer full sentences with punctuation for comments, here and elsewhere: https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar